README.md

```diff
@@ -53,13 +53,13 @@ finch run --rm public.ecr.aws/finch/hello-finch
 
 If you're new to containers, that is so exciting! Give the command above a try after you've installed and initialized Finch. The `run` command pulls an image locally if it's not already present, and then creates and runs a container for you. Note the handy `--rm` option will delete the container instance once it's done executing.
 
-To build an image, try a quick example from the finch client repository.
+To build and run a local image, try a quick example from the finch client repository.
 
 ```sh
-git clone git@github.com:runfinch/finch.git
+git clone https://github.com/runfinch/finch.git
```

Comment:
It may be better to remove the protocol completely and just let it use whatever the user has their git client configured to use.

I've been in some environments where SSH was required due to interference with proxies when using https. I've also been in some environments that would prevent outbound SSH where the configuration needed to be overridden to use https instead.

So basically, the protocol used should be left to the user and not be prescriptive here.

```sh
git clone github.com/runfinch/finch
```

At least that would be my preference.